### ISSUE TITLE
Build tutorial 

### ISSUE THREAD

**Issue by sonbt56** [delebash/mediapipe_hands_web#1]
Very nice work. 

Which wasm build tool are you using? Could you make a simple tutorial how we can archive this build from the C++ code?

**Comment by delebash** [comment]
I did not build the wasm I simply looked at which files where being loaded in the mediapipe web version via dev tools and downloaded them.  They said they are working on an api so it is probably best to wait for that.  This was more of just a fun experiment.

**Comment by prakashjha18** [comment]
hey, I have seen your project but I'm not able to get the 21 points, can you please tell me from where I can get the 21 finger points to use it further?

**Comment by delebash** [comment]
This was just an example project however mediapipe has come out will a full javascript api please see https://google.github.io/mediapipe/getting_started/javascript

**Comment by prakashjha18** [comment]
Yes but is there any way I can get the 21 finger points in your project?
The js api of mediapipe doesn't work with mobile browser

**Comment by delebash** [comment]
Sorry I do not know I did not write this project I just extracted it from mediapipe for the web demo this was before the js version.  If the js version doesn't work on mobile they have ios and android examples.  What fails on the mobile browser with the js version?

**Comment by prakashjha18** [comment]
The camera

**Comment by delebash** [comment]
What do you mean?

**Comment by delebash** [comment]
What error do you get?

**Comment by prakashjha18** [comment]
The camera fails mobile in version of js
https://github.com/google/mediapipe/issues/1459

**Comment by delebash** [comment]
see https://github.com/google/mediapipe/issues/1427.  It is a bug regarding webgl2 and ios safari

**Comment by prakashjha18** [comment]
yes I did had a look at this, but even this does not provide any solution
